blinds/blinds_api.py

```diff
@@ -154,31 +178,41 @@ def getStatus( self ):
     def getSchedule( self ):
         print( "processing request for GET schedule")
         
-        # dummy temporary data
-        # TODO: schedule will likely need to be serialized
-        data = {
-            "schedule" : "SCHEDULE"
-        }
+        try:
+            # data =  BlindsSchedule.toJson( self._blindsSchedule )
+            data = BlindsSchedule.toDict( self._blindsSchedule )
 
-        resp = ( data, RESP_CODES[ "OK" ] )
+            resp = ( data, RESP_CODES[ "OK" ] )
 
-        # TODO: ERROR CASE 
+        # TODO Other error cases
+
+        except Exception as err: # catch all others and return an error message 
+            # TODO : More specialized handling for safety, we don't want just any error messages being spit to the user, for 
+            # now, in the testing phase we return the error 
+            return str( err ), RESP_CODES[ "BAD_REQUEST" ]
 
         return resp
 
     '''
-    API POST request handler for schedule
+    API POST request handler for schedule. For now accept only POST request of a full schedule
```

Comment:
I think we may have to leave it this way until the end of the project. It's not terribly efficient but we are running out of time.

Reply:
Works for me.